app/Utility/CharUtil.php

```diff
@@ -94,26 +109,29 @@ public static function menuBattle(string $mainCharName, string $enemyCharName):
      * @param Character     $enemyChar
      * @return void
      */
-    public static function fight(MainCharacter $mainChar, Character $enemyChar): void
+    public static function fight(MainCharacter $mainChar, Character $enemyChar): ?string
     {
         echo "\n\t    =====================\n";
         $mainChar->attack($enemyChar);
         sleep(1);
         $enemyChar->attack($mainChar);
         echo "\n\t    =====================\n";
         sleep(1);
+
+        if ($mainChar->health == 0) return 'dead';
+        elseif ($enemyChar->health == 0) return 'victory';
+        else return NULL;
     }
 
-    public static function status(MainCharacter $mainChar): void
+    public static function status(Character $character): void
     {
-        $name       = $mainChar->name;
```

Comment:
use getter in accessing the attribute of the class or object. This is not possible if name attribute is private.